src/Sulu/Bundle/CustomUrlBundle/Tests/Functional/Controller/CustomUrlControllerTest.php

```diff
@@ -612,8 +612,8 @@ public function testGet($data, $url)
         $this->assertArrayHasKey('creator', $responseData);
         $this->assertArrayHasKey('changer', $responseData);
 
-        $this->assertEquals(new \DateTime(), new \DateTime($responseData['created']), '', 2);
-        $this->assertEquals(new \DateTime(), new \DateTime($responseData['changed']), '', 2);
+        $this->assertGreaterThanOrEqual(new \DateTime(), new \DateTime($responseData['created']));
+        $this->assertGreaterThanOrEqual(new \DateTime(), new \DateTime($responseData['changed']));
```

Comment:
Is that correct? Shouldn't the datetime be created before the request is sent? Otherwise the code in between runs too long again and might make the test fail.